XiEditor/Base.lproj/MainMenu.xib

```diff
@@ -649,6 +649,25 @@
                         </items>
                     </menu>
                 </menuItem>
+                <menuItem title="Debug" id="r1r-EE-uzA" userLabel="Debug">
+                    <modifierMask key="keyEquivalentModifierMask"/>
+                    <menu key="submenu" title="Debug" id="V9K-nn-2xz">
+                        <items>
+                            <menuItem title="Rewrap" keyEquivalent="" id="eyx-NK-xxg">
```

Comment:
Is it cool to put pseudo-unicode in here? I think I'd be slightly more comfortable with &#xF704;